test/src/test/java/org/corfudb/integration/AbstractIT.java

```diff
@@ -126,7 +178,14 @@ public static String getCorfuServerLogPath(String host, int port) {
      */
     public static void forceShutdownAllCorfuServers() throws IOException, InterruptedException {
         ProcessBuilder builder = new ProcessBuilder();
-        builder.command("sh", "-c", FORCE_KILL_ALL_CORFU_COMMAND);
+        builder.command(SH, HYPHEN_C, FORCE_KILL_ALL_CORFU_COMMAND);
+        Process p = builder.start();
+        p.waitFor();
+    }
+
+    public static void gracefullyShutdownAllCorfuServers() throws IOException, InterruptedException {
```

Comment:
not sure if this new method is needed.. can the existing method shutdownCorfuServer() be used?

Reply:
Done, Thank you.